react/config/rollup.config.js

```diff
@@ -0,0 +1,45 @@
+import html from '@web/rollup-plugin-html';
+import { defineConfig } from 'rollup';
+import babel from '@rollup/plugin-babel';
+import nodeResolve from '@rollup/plugin-node-resolve';
+import commonjs from '@rollup/plugin-commonjs';
+import replace from '@rollup/plugin-replace';
+import json from '@rollup/plugin-json';
+import postcss from 'rollup-plugin-postcss';
+
+export default defineConfig([
+  {
+    input: ['src/*.html'],
+    output: {
+      dir: 'dist',
+      format: "esm",
+      sourcemap: false,
+    },
+    plugins: [
+      json(),
+      replace({
+        preventAssignment: true,
+        'process.env.NODE_ENV': JSON.stringify( 'development' )
```

Comment:
should this always be `development`?

Reply:
This one is up to developer - they can make it as "production" if they always want NODE_ENV to be in that specific state. I will remove it for now actually